redfish-core/lib/log_services.hpp

```diff
@@ -1147,6 +1170,46 @@ inline void requestRoutesSystemLogServiceCollection(App& app)
                     "/xyz/openbmc_project/object_mapper",
                     "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths", "/",
                     0, std::array<const char*, 1>{postCodeIface});
+
+                // Make sure the "HardwareIsolation" service support in the
+                // backend, if it is available then, add into "LogServices".
+                crow::connections::systemBus->async_method_call(
+                    [asyncResp](const boost::system::error_code ec,
+                                const std::vector<std::string>& subtreePath) {
+                        if (ec)
+                        {
+                            BMCWEB_LOG_ERROR
+                                << "D-Bus response error [" << ec.value()
+                                << " : " << ec.message() << "] "
+                                << "when tried to check the HardwareIsolation "
+                                << "service support";
+                            return;
+                        }
+
+                        for (const auto& pathStr : subtreePath)
+                        {
+                            if (pathStr.find("hardware_isolation") !=
+                                std::string::npos)
+                            {
+                                nlohmann::json& logServiceArrayLocal =
+                                    asyncResp->res.jsonValue["Members"];
+                                logServiceArrayLocal.push_back(
+                                    {{"@odata.id",
+                                      "/redfish/v1/Systems/system/"
+                                      "LogServices/HardwareIsolation"}});
```

Comment:
Even when the link isn't there, you can still get to the path... :/ 
line 3481 / 4402
I'm not sure this makes sense, I think it would be better to just always show it and when HardwareIsolation is empty, just show an list

Reply:
I don't think PostCode did this correctly :( 
Can we just show this always, I think that is the simplest solution?

Reply:
What happens if we don't have `HardwareIsolation` feature backend support in the system? still, we should show/allow the URI (/redfish/v1/Systems/system/LogServices/HardwareIsolation)?